ap_src/ap_app/static/js/button_spinner.js

```diff
@@ -0,0 +1,42 @@
+/*Logic for spinner for create team button -KJ*/
```

Comment:
You can remove your initials from comments. We can use git blame to show us who changed each line in a file if we wanted to.